### ISSUE TITLE
Periodically, view with iCloud3 event log card appears blank

### ISSUE THREAD

**Issue by cerebrate** [gcobb321/icloud3#21]
i.e., none of the cards are rendered at all.

On occasions when this happens, the following script error shows up in the browser console:

```
icloud3-event-log-card.js:589 Uncaught TypeError: Cannot read property 'slice' of undefined
    at HTMLElement._setupEventLogTable (icloud3-event-log-card.js:589)
    at HTMLElement.set hass [as hass] (icloud3-event-log-card.js:453)
    at HTMLElement.value (hui-view.ts:61)
    at HTMLElement.value (hui-view.ts:337)
    at HTMLElement.t.addEventListener.once (hui-view.ts:68)
    at r (fire_event.ts:76)
    at create-card-element.ts:111
```

using version 2.0.5.

**Comment by gcobb321** [comment]
Each entry on the log has a text message. The error at :589/slice indicates that the text message is blank for some unknown reason. I'll add some error catching code on the next iCloud3 update to make sure that there are no blank messages for the event log.

The next version of iCloud3 that is under development is in the [iCloud3 GitHub repository here](https://github.com/gcobb321/icloud3/tree/master/development). Download it and see if this fixes the error.

**Comment by cerebrate** [comment]
Thanks. I've installed that. Everything's fine so far, but since this error sometimes takes a while to manifest, I'll give it more time and report back if it's still working.

Meanwhile, on another data point, while the error shows up on all my usual browsers (Edge (Chromium) on Windows, Edge on iOS, and Safari on iOS, as well as the iOS HA app), I noticed today that my smart mirror, which displays the HA views in rotation using Chromium on Raspbian, didn't seem to suffer from it even while all the others did.
